src/main/configSchema.ts

```diff
@@ -116,6 +118,12 @@ export const configSchema = {
     minimum: 15,
     maximum: 60,
   },
+  obsForceMono: {
+    description:
+      'Whether to force the audio of your input device to mono. Enable if your microphone audio is only playing out of  one stereo channel.',
```

Comment:
I think you can drop the "Whether to" here.
Also double space between "of  one"